src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -664,7 +671,26 @@ protected void updateRenderMetaData()
     @Override
     protected boolean neededForWorker(@Nullable final ItemStack stack)
     {
-        return isStackAxe(stack) || isStackSapling(stack);
+        return isStackAxe(stack);
+    }
+
+
+     /**
+     * Override this method if you want to keep an amount of items in inventory.
+     * When the inventory is full, everything get's dumped into the building chest.
+     * But you can use this method to hold some stacks back.
+     *
+     * @return a list of objects which should be kept.
+     */
+    protected Map<ItemStorage, Integer> needXForWorker()
+    {
+        Map<ItemStorage, Integer> keepX = new HashMap<>();
+        ItemStack stack = new ItemStack(Blocks.SAPLING);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'stack' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)